corgi/api/serializers.py

```diff
@@ -413,7 +413,7 @@ def get_provides(instance: Component) -> list[dict[str, str]]:
     @staticmethod
     def get_sources(instance: Component) -> list[dict[str, str]]:
         return get_component_data_list(
-            instance.sources.values_list("purl", flat=True).using("read_only")
+            instance.sources.values_list("purl", flat=True).using("read_only").iterator()
```

Comment:
Curious if `prefetch_related` on the ManyToMany could help reducing number of queries in this specific case?

Reply:
Thanks